src/components/Store/StoreForm.tsx

```
@@ -0,0 +1,158 @@
+"use client"
+
+import LoadingErrorContainer from "@/components/feedback/LoadingErrorContainer"
+import { ValidationErrorList } from "@/components/feedback/validationErrorList"
+import { StoreFormInputText } from "@/components/Store/StoreFormInputText"
+import { StoreRegisterToppingCallsCheck } from "@/components/toppingCallOptions/StoreRegisterToppingCalls"
+import { StoreSubmitButton } from "@/components/Store/StoreSubmitButton"
+import StoreSwitch from "@/components/Store/StoreSwitch"
+import { useUpdateStore } from "@/hooks/api/useStores"
+import { useStoreForm } from "@/hooks/useStoreForm"
+import { StoreFormInput } from "@/validations/store"
+import { AccessTime, EventBusy, ExpandMore, Map, Store } from "@mui/icons-material"
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)